.github/workflows/check_format_code.yml

```diff
@@ -9,6 +9,10 @@ jobs:
     - uses: actions/checkout@v2
     - name: Run bootstrap
       run: cd tools/ && ./bootstrap
+    - name: Install prototool
+      run: brew install prototool  
```

Comment:
Huh, I didn't know Homebrew exists on Linux too! And it's preinstalled on GitHub's Ubuntu? Cool.